a10_octavia/common/config_options.py

```diff
@@ -33,6 +33,9 @@
 
 
 A10_GLOBAL_OPTS = [
+    cfg.StrOpt('network_type',
+               default='flat',
```

Comment:
You can add choices here. like this 
` choices=a10constants.SUPPORTED_NETWORK_TYPE,`

And then can add `SUPPORTED_NETWORK_TYPE` in `a10constants.py`  like 
`SUPPORTED_NETWORK_TYPE = (FLAT, VLAN)`
`VLAN = 'vlan'`
`FLAT= 'flat'`